pkg/bootloader/grub.go

```diff
@@ -37,11 +40,11 @@ type Grub struct {
 }
 
 type grubBootEntry struct {
-	linux       string
-	initrd      string
-	cmdline     string
-	displayName string
-	id          string
+	Linux       string
+	Initrd      string
+	CmdLine     string
+	DisplayName string
+	ID          string
```

Comment:
These values are made public because this struct is then passed to the grub config template files hence this requires to be public so the templating library can actually access them.

We could eventually keep them private and add a method to grubBootEntry struct to return a map with the values, that could also work for the template. I have no strong opinion.